include/pru/am335x.h

```diff
@@ -85,7 +90,7 @@
 
 struct __PRU_CONTROL_S {
 	const volatile unsigned int SOFT_RST_N		: 1;
-	volatile unsigned int ENABLE			: 1;
+	volatile unsigned int EN				: 1;
```

Comment:
In " SPRUHF8A – May 2012 – Revised June 2013", page 76, bit field is named ENABLE.

Similarly for all bit field naming updates bellow.

Reply:
I used this document: 
" Literature Number: SPRUH73Q
October 2011 – Revised December 2019 "